x-pack/legacy/plugins/actions/server/builtin_action_types/email.ts

```diff
@@ -102,7 +102,9 @@ function validateParams(paramsObject: any): string | void {
 export function getActionType({ logger }: { logger: Logger }): ActionType {
   return {
     id: '.email',
-    name: 'email',
+    name: i18n.translate('xpack.actions.builtin.email.name', {
```

Comment:
```suggestion
    name: i18n.translate('xpack.actions.builtin.emailTitle', {
```

Reply:
Fixed in https://github.com/elastic/kibana/pull/47843/commits/85578a4fc084818300dbc9fe472dab2050dce9a5.